hardhat.config.ts

```diff
@@ -0,0 +1,39 @@
+import "@nomiclabs/hardhat-ethers"
+import "hardhat-jest-plugin"
+import "hardhat-deploy"
+import { readdir, readFile, writeFile } from "fs/promises"
+import { HardhatUserConfig } from "hardhat/types"
+import { task } from "hardhat/config"
+
+task(
+  "convert",
+  "Convert all json abis to human readable format",
+  async (_, hre) => {
```

Comment:
This hre argument is a bit cryptic.
Maybe extend the name and/or add a type annotation to it.
Alternavely you could also unpack it here directly as `{ ethers }` since we are only using the `ethers` member here

Reply:
hre is what they use in the docs,  but I see your point. I think unpacking it is a good idea though for the reason you state. I'll modify